src/lib.rs

```diff
@@ -160,7 +187,7 @@ impl ProcessManager {
 			.envs(process_data.process.env.clone())
 			.stdout(Stdio::piped())
 			.stderr(Stdio::piped())
-			.stdin(Stdio::null())
+			.stdin(Stdio::piped())
```

Comment:
So, this always uses a pipe for stdin?  Will uses of this always/usually want a pipe?

I guess it isn't too much of a problem, though,  to have a pipe that nothing writes to. Insignificantly more overhead. (Unlike an stdout/stderr pipe, which needs to be read or the process will block when the pipe buffer is filled.)

Reply:
Most uses of this probably won't usually want a pipe. I can refactor this to make it optional?

Reply:
In principle I think it shouldn't be creating a pipe when it normally isn't needed, but that may be simpler in some ways.